app/models/concerns/duplicate_roster_entries.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module DuplicateRosterEntries
+  extend ActiveSupport::Concern
+
+  module ClassMethods
+    # rubocop:disable Metrics/MethodLength
+    def add_suffix_to_duplicates(identifiers: [], roster_entries: [])
```

Comment:
What is the purpose of `roster_entries:` here? It looks like this method mutates the argument which would be unexpected for a method without a `!` in the signature. Do we need to mutate anything here?

Reply:
The reason I added the field `roster_entries` was to pass in any existing roster entries that are on the roster. This way we can take them into consideration for duplicates. This is mainly only used for adding more students into the roster or syncing from integrations. 

I just changed the code a little bit, so that we are not mutating `roster_entries`. Also that field is now renamed to `existing_roster_entries` so that is is more clear on what that variable does. Instead of mutating, I added `list_of_identifiers` and `existing_roster_entries` together so we can also take into consideration the new identifiers that are being added.